<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataGenerator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#fff;</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="bouton1">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
padding:4px;
border:solid;
color: white;
font-size: 12px;
font-weight: 400;
background: #34495e;
}

QPushButton:pressed {
background: #2D3F51;
color: white;
}

QPushButton:hover {
background: #1C2E40;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Ouvrir un fichier...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bouton6">
        <property name="styleSheet">
         <string notr="true">QPushButton {
padding:4px;
border:solid;
color: white;
font-size: 12px;
font-weight: 400;
background: #34495e;
}

QPushButton:pressed {
background: #2D3F51;
color: white;
}

QPushButton:hover {
background: #1C2E40;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="listWidget1">
        <property name="styleSheet">
         <string notr="true">background-color:#fff;
border:1px solid #ccc;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget2">
        <property name="styleSheet">
         <string notr="true">background-color:#fff;
border:1px solid #ccc;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget3">
        <property name="styleSheet">
         <string notr="true">background-color:#fff;
border:1px solid #ccc;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEdit1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Nombre de ligne</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#fff;
border:1px solid #ccc;</string>
        </property>
        <item>
         <property name="text">
          <string>Oracle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MySQL</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="bouton2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
padding:4px;
border:solid;
color: white;
font-size: 12px;
font-weight: 400;
background: #34495e;
}

QPushButton:pressed {
background: #2D3F51;
color: white;
}

QPushButton:hover {
background: #1C2E40;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Générer les valeurs aléatoires</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bouton3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
padding:4px;
border:solid;
color: white;
font-size: 12px;
font-weight: 400;
background: #34495e;
}

QPushButton:pressed {
background: #2D3F51;
color: white;
}

QPushButton:hover {
background: #1C2E40;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Sauvegarder le fichier SQL</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
color: white;
font-size: 12px;
font-weight: 400;
background: #34495e;
}

QMenuBar::item {
color: white;
font-size: 12px;
font-weight: 400;
background: #34495e;
}

QMenuBar::item:selected {
background: #1C2E40;
color: white;
}</string>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="styleSheet">
     <string notr="true">QMenu {
color: white;
font-size: 12px;
font-weight: 400;
background: #34495e;
}

QMenu::item {
color: white;
font-size: 12px;
font-weight: 400;
background: #34495e;
}

QMenu::item:selected {
background: #1C2E40;
color: white;
}</string>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionNewList"/>
    <addaction name="actionUpdateList"/>
    <addaction name="actionDeleteList"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionNewList">
   <property name="text">
    <string>Créer une liste...</string>
   </property>
  </action>
  <action name="actionUpdateList">
   <property name="text">
    <string>Modifier une liste...</string>
   </property>
  </action>
  <action name="actionDeleteList">
   <property name="text">
    <string>Supprimer une liste...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
